pkg/process/procutil/process_linux.go

```diff
@@ -447,6 +454,56 @@ func (p *Probe) parseStatContent(statContent []byte, sInfo *statInfo, pid int32,
 	return sInfo
 }
 
+// parseStatm gets memory info from /proc/(pid)/statm
+func (p *Probe) parseStatm(pidPath string) *MemoryInfoExStat {
+	path := filepath.Join(pidPath, "statm")
+	var err error
+
+	memInfoEx := &MemoryInfoExStat{}
+
+	contents, err := ioutil.ReadFile(path)
+	if err != nil {
+		return memInfoEx
+	}
+	fields := strings.Split(string(contents), " ")
```

Comment:
Why do we split the file here by space whereas we need to parse `stat` file character by character? It's just because `stat` is a smaller file so we don't have a lot of performance impact by using `strings.Split()`?